app/Repositories/PhotoRepository.php

```diff
@@ -44,7 +44,11 @@ public function create(array $data, $signupId)
     {
         $postEvent = Event::create($data);
 
-        $fileUrl = $this->aws->storeImage($data['file'], $data['campaign_id']);
+        if (array_key_exists('campaign_id', $data)) {
+            $fileUrl = $this->aws->storeImage($data['file'], $data['campaign_id']);
+        } else {
+            $fileUrl = $this->aws->storeImage($data['file'], $signupId);
+        }
```

Comment:
hmm thanks for pointing this out! I think we should somehow standardize file names (That way it is easier to search if we ever need to) I don't really care what the standard is, but it should be something we can easily identify, maybe we just use `signupId` as the standard instead of `campaign_id`?